Classes/Issues/GithubClient+Issues.swift

```diff
@@ -289,13 +289,21 @@ extension GithubClient {
             return
         }
 
-        // https://developer.github.com/v3/repos/collaborators/#check-if-a-user-is-a-collaborator
+        // https://developer.github.com/v3/repos/collaborators/#review-a-users-permission-level
         request(Request(
-            path: "repos/\(owner)/\(repo)/collaborators/\(viewer)",
+            path: "repos/\(owner)/\(repo)/collaborators/\(viewer)/permission",
             headers: ["Accept": "application/vnd.github.hellcat-preview+json"],
             completion: { (response, _) in
-                // documentation states that collab = 204
-                completion(.success(response.response?.statusCode == 204))
+                let statusCode = response.response?.statusCode
+                if statusCode == 200,
+                    let json = response.value as? [String: Any],
+                    let permission = json["permission"] as? String {
```

Comment:
Super minor but is it worth loading these permissions into an enum and passing that value through to be handled else where?

The enum could then have a `canManageIssue: Bool` which returns true if `self == .admin || self == .write`.

Just thinking ahead we may want to use the user's permission for other things and makes it a bit more flexible without locking this down to strictly this one use case?

Reply:
I was wondering this too. Doesn't hurt either way. I'll add this!